### ISSUE TITLE
How to define alternate?

### ISSUE THREAD

**Issue by a0s** [jbaubree/vite-plugin-sitemap#64]
Hi! Thanks for the nice plugin! I am trying to build sitemap.xml, but i need alternate links for all languages my site supports. How can i do this using dynamicRoutes? Example:
```xml

<?xml version="1.0" encoding="UTF-8"?>
  <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
  <url>
  <loc>https://www.somesite.com/</loc>
  <xhtml:link rel="alternate" hreflang="x-default" href="https://www.somesite.com/"/>
  <xhtml:link rel="alternate" hreflang="en" href="https://www.somesite.com/"/>
  <xhtml:link rel="alternate" hreflang="es" href="https://www.somesite.com/es/"/>
  <xhtml:link rel="alternate" hreflang="pt" href="https://www.somesite.com/pt/"/>
  <xhtml:link rel="alternate" hreflang="fr" href="https://www.somesite.com/fr/"/>
  <xhtml:link rel="alternate" hreflang="de" href="https://www.somesite.com/de/"/>
  <xhtml:link rel="alternate" hreflang="it" href="https://www.somesite.com/it/"/>
  <xhtml:link rel="alternate" hreflang="ja" href="https://www.somesite.com/ja/"/>
  <xhtml:link rel="alternate" hreflang="ko" href="https://www.somesite.com/ko/"/>
  <xhtml:link rel="alternate" hreflang="ru" href="https://www.somesite.com/ru/"/>
  <xhtml:link rel="alternate" hreflang="uk" href="https://www.somesite.com/uk/"/>
  <xhtml:link rel="alternate" hreflang="tr" href="https://www.somesite.com/tr/"/>
  <xhtml:link rel="alternate" hreflang="pl" href="https://www.somesite.com/pl/"/>
  <xhtml:link rel="alternate" hreflang="nl" href="https://www.somesite.com/nl/"/>
  <xhtml:link rel="alternate" hreflang="no" href="https://www.somesite.com/no/"/>
  <changefreq>daily</changefreq>
  </url>
```

**Comment by a0s** [comment]
Thanks for response. After several hours of experiments i decided to rewrite this functionality from scratch https://gist.github.com/a0s/a15f0d3dee14d5c4457dc2dc77657207

**Comment by jbaubree** [comment]
@a0s FYI, i18n support is now available in `v0.6.0`, you can follow [README](https://github.com/jbaubree/vite-plugin-sitemap?tab=readme-ov-file#i18n)
Feature commit: https://github.com/jbaubree/sitemap-ts/commit/889b2731c36b8507bf35044f95e99e5ed0ef798e
